es/faq.html

```diff
@@ -130,52 +130,269 @@ <h6 class="accordion-toggle border-top pt-3 pb-2 green" id="dao-for-traders">{{
 
   <h6 class="accordion-toggle border-top pt-3 pb-2 green" id="without-bsq">{{ item.hUseBSQ }}</h6>
   <div class="accordion-content pb-5 col-sm-12 col-md-8 ml-0 pl-sm-0 pl-md-0 px-0">
-      {{ item.pUseBSQ }}
+<h1>{{ item.hFAQ }}</h1>
```

Comment:
Looks like lines 11 through 132 are included by mistake?

Reply:
Yes it was a mistake. Fixed.